addons/binding/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/OpenThermGatewaySocketConnector.java

```diff
@@ -81,7 +99,7 @@ public void sendCommand(CommandType commandType, String message) {
 
             String msg = command.getMessage(message);
 
-            // msg = msg + "\r\n";
+            msg = msg + "\r\n";
```

Comment:
Hi Arjen, 

I was busy creating a pull request, but I just saw your commit:
 println seems os specific, so it's not always terminated properly. The solution which works for me is changing println to:
writer.printf("%s\r\n", msg);
This fixed the setpoint problem on my setup (openhabian on pi3+ nodoshop otgw/nodeMCU) 

Hope this helps